packages/ella/src/general/index.ts

```diff
@@ -834,4 +834,39 @@ export const parseJSON = (parameter:string, fallback='', reviver?: (this: any, k
   } catch (exception) {
     return fallback;
   }
-};
\ No newline at end of file
+};
+
+
+/*
+ * Strictly check if all values in an object is 0.
+ *
+ * @param object
+ * @returns boolean
+ *
+ * @example
+ * ```
+ * isAllObjectValuesZero({ key1: 0, key2: 0, key3: 0 }) // true
+ * isAllObjectValuesZero({ key1: '0', key2: 0, key3: 0 }) // false
+ * isAllObjectValuesZero({ key1: 'some value', key2: 0, key3: 0 }) // false
+ * isAllObjectValuesZero(null) // true
+ * isAllObjectValuesZero(undefined) // true
+ * isAllObjectValuesZero({}) // true
+ *
+ * Behavior with Array(Not expected in the args) -
```

Comment:
No point in sharing behavior with Array because you are not accepting array as param. Either remove array examples or modify the name and definition of the method to handle array as well.